api.yaml

```diff
@@ -972,7 +972,7 @@ components:
     AccountOwner:
       type: object
       properties:
-        accountId:
+        accountId: # TODO add specific type once we are sure we return it correctly
```

Comment:
I am not 100% sure we stopped returning null here, so not adding the validation here.

Reply:
Yes, I definitely wouldn't touch the account owner stuff for lookups yet.